README.md

```diff
@@ -82,6 +82,58 @@ Once you have written your routes, a good step is to run the `mix validate_route
 task, which will load the routes for the current `MIX_ENV` and check them all
 for correctness.
 
+### Cross-Origin Resource Sharing
+
+Spacesuit provides middleware that implements Cross-Origin Resource Sharing (CORS).
+
+CORS is a protocol that allows web applications to make requests from a browser
+across different domains. A full specification can be found [here](https://www.w3.org/TR/cors/).
+
+#### Configuring the CORS filter
+
+The middleware can be configured within the routes configuration file. For a full
+listing of configuration options, see the [config.exs](blob/master/config/config.exs#L70-L84).
+
+The available options include:
```

Comment:
think @relistan prefers using the commented out code in the `config.exs` to serve as example configuration; i've called that out just above this.